### ISSUE TITLE
Docker Container latest build failing to start

### ISSUE THREAD

**Issue by 1activegeek** [unifi-poller/unifi-poller#165]
Latest update to the docker container is breaking the container. Fails to start using latest, stable, or master tags. All produce the same error message:

```
2019/12/29 22:47:26 [INFO] Loading Configuration File: /etc/unifi-poller/up.conf
2019/12/29 22:47:26 [INFO] UniFi Poller v2.0.0-alfalfa1-695 Starting Up! PID: 1
panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x88 pc=0x869b2c]

goroutine 1 [running]:
github.com/unifi-poller/inputunifi.(*InputUnifi).Initialize(0xc00015a030, 0xacbf00, 0xc00001e280, 0xc000168140, 0xc0000c2030)
/go/pkg/mod/github.com/unifi-poller/inputunifi@v0.0.1/interface.go:17 +0x3c
github.com/unifi-poller/poller.(*UnifiPoller).InitializeInputs(0xc00001e280, 0x0, 0x0)
/go/pkg/mod/github.com/unifi-poller/poller@v0.0.1/inputs.go:73 +0xe7
github.com/unifi-poller/poller.(*UnifiPoller).Run(0xc00001e280, 0x0, 0x0)
/go/pkg/mod/github.com/unifi-poller/poller@v0.0.1/start.go:78 +0x11e
github.com/unifi-poller/poller.(*UnifiPoller).Start(0xc00001e280, 0xc00015a180, 0x40576f)
/go/pkg/mod/github.com/unifi-poller/poller@v0.0.1/start.go:40 +0x124
main.main()
/unifi-poller/main.go:16 +0xb7
```

As far as I can tell this is isolated to the latest push to these tags. Dropping back to `2.0.0-alpha1` works without an issue. If this is not affecting everyone, I'm happy to help troubleshoot what in my configs could be causing this. No changes on my side recently though, so I'm not sure what could be causing this trouble.

**Comment by davidnewhall** [comment]
The `latest` tag was updated with a beta release. It broke a lot of people :(

Please use tag `1.6.3` for now. Instructions for upgrading will be coming soon.

**Comment by 1activegeek** [comment]
Thanks David - just a point of caution, it affected more than just the latest tag. It would appear your stable and latest both had the same code based on the matching digests (as well as master). Not sure if there is someone else managing the docker container, but I'd highly suggest the code being used is separated so that anyone using the stable tag, is not receiving beta code as it appears happened here. I was on the stable branch, and only by investigating the tags, associated digests, and update timeframes did I find that the 2.0.0 tag was working. I did test 1.6.3 as well and have since rolled back.

<img width="213" alt="image" src="https://user-images.githubusercontent.com/6810720/71564052-ff36c900-2a67-11ea-8c74-e8825acadb95.png">
<img width="214" alt="image" src="https://user-images.githubusercontent.com/6810720/71564057-0cec4e80-2a68-11ea-825a-ff45ff94a6c5.png">

**Comment by davidnewhall** [comment]
The build scripts can't currently identify a beta/alpha tag, and treat it as any normal tagged release. I need to fix it; I did not want the latest/stable docker tags updated from that git tag. 

More info: I recently linked the latest and stable tags and moved master updates to the `master` tag. The `latest` tag should now also be `stable`, but this was just a screw-up. 

I'll see if I can do some manual magic to fix those docker tags now.

**Comment by davidnewhall** [comment]
Additionally, I'll get the docs for v2 added/updated this week, so upgrading should be less painful.

**Comment by davidnewhall** [comment]
I've re-pushed the 1.6.3 images to the `latest` and `stable` tags. This should be fixed (reverted). I'll leave this issue open until I fix the code that caused this to happen. Please let me know if `latest` works now, if you dare trust me. haha thanks!

**Comment by 1activegeek** [comment]
> I've re-pushed the 1.6.3 images to the `latest` and `stable` tags. This should be fixed (reverted). I'll leave this issue open until I fix the code that caused this to happen. Please let me know if `latest` works now, if you dare trust me. haha thanks!

Yup, looks good. Just altered to use latest and it looks to be good. Thanks for the quick revert.  Looking forward to 2.0!

**Comment by davidnewhall** [comment]
The bug that caused the crash (nil pointer dereference) was also fixed, here: 
https://github.com/unifi-poller/inputunifi/compare/v0.0.2...v0.0.3

**Comment by 1activegeek** [comment]
SWEET! 2.0 ready now?! 😆

**Comment by davidnewhall** [comment]
Yeah, it was ready when I made the `2.0.0-alpha` release. haha I just didn't have any docs yet. The important part is passing in your controller config properly (using new up.conf format or new env variables).

I've updated all the docs, and they are currently in the testing wiki: https://github.com/unifi-poller/wiki/wiki/Configuration#quick-start

The v2.0.0 dashboards are in the dashboards repo: https://github.com/unifi-poller/dashboards/tree/master/v2.0.0 (1 is still not complete or uploaded)
